File: 65 Valid Number.cpp
class Solution {
public:
    bool isNumber(string s) {
        //超級慢
        regex e("^ *[+-]?(([0-9]+\\.?[0-9]*)|(\\.[0-9]+))(e[+-]?[0-9]+)? *$");
        return std::regex_match (s, e);
    }
};

File: 65 Valid Number.py
class Solution:
    def isNumber(self, s):
        import re
        re = re.compile("^[+-]?(([0-9]+\\.?[0-9]*)|(\\.[0-9]+))(e[+-]?[0-9]+)?$")
        result = re.search(s.strip())
        return result != None